app/controllers/auth/sso_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Auth
+  class SsoController < Api::V1::BaseController
+    include ActionController::Cookies
+
```

Comment:
This controller should assert the SSO entitlement, i.e. it should only be available in Keygen EE.